config/application.rb

```diff
@@ -43,11 +43,13 @@ class Application < Rails::Application
     # Applying the patch for CVE-2022-32224 broke YAML deserialization because some classes are disallowed in the serialized YAML
     # NOTE: Symbol was later added to enabled classes by default, see https://github.com/rails/rails/pull/45584,
     # it was added to Rails 6.0.6, 6.1.7, 7.0.4
-    config.active_record.yaml_column_permitted_classes = [Symbol, Time, Date, BigDecimal, OpenStruct,
-                                                          ActionController::Parameters,
-                                                          ActiveSupport::TimeWithZone,
-                                                          ActiveSupport::TimeZone,
-                                                          ActiveSupport::HashWithIndifferentAccess]
+    config.active_record.yaml_column_permitted_classes = %w[Symbol Time Date BigDecimal OpenStruct
+                                                            Proxy::PolicyConfig PoliciesConfig
```

Comment:
What if you refer to `Proxy::PoliciesConfig` here? won't it work?

Reply:
it is when audited does marshaling/unmarshaling, I don't like how it works but this is how it presently works

Reply:
I meant, here you are using `PoliciesConfig` but that's the alias we'd like to remove. I think you could use `Proxy::PoliciesConfig` and YAML serialization should work the same.